ExampleApp/ExampleApp/Screens/Characteristics/View/UITableViewCell/CharacteristicsTableViewCell.swift

```diff
@@ -0,0 +1,61 @@
+import RxBluetoothKit
+import UIKit
+
+class CharacterisiticsTableViewCell: UITableViewCell {
+
+    let uuidLabel: UILabel = UILabel(style: Stylesheet.Commons.titleLabel)
+
+    let isNotyfingLabel: UILabel = UILabel(style: Stylesheet.Commons.titleLabel)
+
+    let valueLabel: UILabel = UILabel(style: Stylesheet.Commons.titleLabel)
+
+    override init(style: UITableViewCellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        backgroundColor = .white
+        setConstraints()
+    }
+
+    @available(*, unavailable)
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func prepareForReuse() {
+        super.prepareForReuse()
+        uuidLabel.text = nil
+        isNotyfingLabel.text = nil
+        valueLabel.text = nil
+    }
+    
+    private func setConstraints() {
+        [uuidLabel, isNotyfingLabel, valueLabel].forEach {
+            $0.translatesAutoresizingMaskIntoConstraints = false
+            addSubview($0)
+        }
```

Comment:
It's a common pattern that you use. Maybe it's worth an extension on the `UIView` type? `addSubviews(_ views: [UIView])` method